harness/determined/deploy/local/cluster_utils.py

```diff
@@ -15,6 +16,9 @@
 from determined.deploy.errors import MasterTimeoutExpired
 from determined.deploy.healthcheck import wait_for_master
 
+AGENT_NAME_DEFAULT = "det-agent-<hostname>"
```

Comment:
I think you can assign this a value like `f"det-agent-{socket.gethostname()}"` like down in line 366

Reply:
Ah listen to Ilia instead 😅